spec/models/game_spec.rb

```diff
@@ -22,4 +22,13 @@
       expect(result).to eq false
     end
   end
+
+  describe 'available' do
+    let(:game) {FactoryBot.create :game}
+
+    it 'should show available games, which are games with total_players = 1' do
+      result = Game.available
+      expect(Game.available.count).to eq(1)
```

Comment:
Lines 30 and 31 could be refactored a bit. Perhaps 30: `result = Game.available.count` and 31: `expect(result).to eq(1)`?